### DISCUSSION TITLE
Linux to Linux: wrong keyboard layout

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk#8922] [Q&A]
I'v Linux on both ends, client and control. I use QUERTZ keyboard layout on control. When I connect to an other Linux system I get a QEURTY keyboard layout. With Window and Mac OS clients the keyboard is mapped correctly.

How to fix this?

**Comment by fufesou** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#keyboard-translation-modes

Do you use Wayland as the control side? If so, only "map mode" is supported for now.

**Reply by mahescho** [reply]
Yes, the control side is Wayland. The only mode offered is "map mode". The Linux client is X11. Keyboard mapping does not work as expected.
